### PR TITLE
use alt z for alternative s1 binomial test

### PR DESCRIPTION
**Pull request by GiovanniVolta**
_Before you submit this PR: make sure to put all operations-related information in a wiki-note, a PR should be about code and is publicly accessible_

## What does the code in this PR do / what does it improve?
Instead of using 'z' for the alternative s1 binomial test, we use 'alt_z'. It is computed in the same way that z_obs i computed in [event_processing](https://github.com/XENONnT/straxen/blob/c003ee1cc8fc40ec74ea32dfb0e24a0414783951/straxen/plugins/event_processing.py#L507).

## Can you briefly describe how it works?
The important lines are here below. I would say that they speak for themselves:

        alt_drift_time = events['s2_center_time'] - events['alt_s1_center_time']
        alt_z = - self.electron_drift_velocity * (alt_drift_time - self.electron_drift_time_gate)
        alt_position = np.vstack([events['x'], events['y'], alt_z]).T     
        alt_aft_prob = self.s1_aft_map(alt_position)

Thanks, @a1exndr, for the found the bug.

_Please include the following if applicable:_
  - [ ] _Update the docstring(s)_
  - [ ] _Update the documentation_
  - [ ] _Tests to check the (new) code is working as desired._
  - [ ] _Does it solve one of the open issues on github?_

### CODE REVIEW

**Comment by JoranAngevaare** [comment]
Thanks @a1exndr and @GiovanniVolta !

File: straxen/plugins/event_patternfit.py [hunk 1]
```diff
@@ -133,6 +142,11 @@ def compute(self, events):
         positions = np.vstack([events['x'], events['y'], events['z']]).T
         aft_prob = self.s1_aft_map(positions)
         
+        alt_drift_time = events['s2_center_time']-events['alt_s1_center_time']
+        alt_z = -self.electron_drift_velocity*(alt_drift_time-self.electron_drift_time_gate)
+        alt_position = np.vstack([events['x'], events['y'], alt_z]).T
```
Comment (JoranAngevaare) [inline straxen/plugins/event_patternfit.py:147]: Shouldn't you have alt_* here?

Comment (GiovanniVolta) [reply straxen/plugins/event_patternfit.py]: Actually no, it has assumed that the alternative s1 happens in the same corrected (x;y) position of the main s1. Furthermore, we don't have corrected alt_x and alt_y.

**Review by JoranAngevaare — APPROVED** [review]
From my side it looks good, thanks again

**Comment by GiovanniVolta** [comment]
I would like feedback from some AC. Just to be sure. 
I would like also take the chance to change the default map in event_patternfit according to [PR#115](https://github.com/XENONnT/private_nt_aux_files/pull/115) in _private_nt_aux_files_. 
Please let me know if you have any other check in mind that I should do.

**Review by kdund — APPROVED** [review]
This makes sense to me, thanks @GiovanniVolta 
(I'm going to assume that all  checks failing are functions of missing corrections available for the testing script?)

**Comment by kdund** [comment]
> This makes sense to me, thanks @GiovanniVolta (I'm going to assume that all checks failing are functions of missing corrections available for the testing script?)

Oh, and the branch should also be updated with main before merging back :)

**Comment by GiovanniVolta** [comment]
@kdund exactly! We should first merge the privater_nt_aux_files PR and let the tests do their stuff

**Comment by WenzDaniel** [comment]
One question would you not also compute it for alt_s1 paired with alt_s2 to have all sensible permutations?

**Comment by GiovanniVolta** [comment]
> 
> 
> One question would you not also compute it for alt_s1 paired with alt_s2 to have all sensible permutations?

It would be nice, but we don't correct alt_x and alt_y. Hence, the binomial test would not be reliable

**Comment by GiovanniVolta** [comment]
Hi All, could we proceed with the merging?

**Comment by WenzDaniel** [comment]
Sure since it has been already approved.

**Pull request merged**